main/ServiceManager/ServiceManager.cpp

```diff
@@ -128,6 +129,10 @@ void ServiceMangerTaskInit()
  */
 void ServiceMangerInit()
 {
+    if (SharedBusInit())
```

Comment:
use `{ ... }` for all `if` statements, even if is has one line of code inside

Reply:
fixed.